pom.xml

```diff
@@ -42,6 +42,8 @@
   <url>http://blueflood.io</url>
 
   <properties>
+    <VERSION_SUFFIX>0-SNAPSHOT</VERSION_SUFFIX>
+    <BLUEFLOOD_VERSION>2.0.${VERSION_SUFFIX}</BLUEFLOOD_VERSION>
     <maven.javadoc.skip>true</maven.javadoc.skip>
```

Comment:
That was option #2 in my [write-up](https://one.rackspace.com/pages/viewpage.action?title=Maven+Versioning&spaceKey=cloudmetrics#MavenVersioning-ProposedSolutions). I decided not to go that route because it requires editing the pom files. Editing them then leaves us with the question of what to do with the edited files: 
* We can't commit them to master, because we're trying to run this off of new commits to master, which would cause an infinite loop
* We could create a commit as part of a release-specific branch, but we don't really support old versions of the software. We don't backport bug fixes or anything. We just generally forget about them.
* We could create a commit and then just tag it. This doesn't really buy us anything either. It just leaves a lot of dangling, tagged commits in the repo.